fpdf/output.py

```diff
@@ -553,7 +553,10 @@ def _add_fonts(self):
                     LOGGER.warning(
                         "Font %s is missing the following glyphs: %s",
                         fontname,
-                        ", ".join(chr(x) for x in font.missing_glyphs),
+                        ", ".join(
+                            f"{chr(x)} ({chr(x).encode('unicode-escape').decode()})"
```

Comment:
Maybe put some kind of quotes around the literal character, to make it obvious when it is a non-printing or whitespace character?

Reply:
Excellent idea!
I just implemented that.